src/index.js

```diff
@@ -0,0 +1,145 @@
+let temperature = 48;
+
+const tempValue = document.getElementById('tempValue');
+const increaseTempControl = document.getElementById('increaseTempControl');
+const decreaseTempControl = document.getElementById('decreaseTempControl');
+const landScape = document.getElementById('landscape');
+const cityNameInput = document.getElementById('cityNameInput');
+const cityNameReset = document.getElementById('cityNameReset');
+const headerCityName = document.getElementById('headerCityName');
+const getRealTimeTempButton = document.getElementById('getRealTimeTempButton');
+
+//Update the sky options when we choose different sky and change the color background in garden
+const skyOptions = {
+    'sunny': {
+        'text': "☁️ ☁️ ☁️ ☀️ ☁️ ☁️",
+        'backgroundColor': "#DFFFFF",
+    },
+    'cloudy': {
+        'text': "☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️",
+        'backgroundColor': "#D3D3D3",
+    },
+    'rainy': {
+        'text': "🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧",
+        'backgroundColor': "#B0C3DF",
+    },
+    'snowy': {
+        'text': "🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨",
+        'backgroundColor': "#ADD9E6",
+    },
+};
+const skySelect = document.getElementById("skySelect");
+const skyDisplay = document.getElementById("skyDisplay")
+const gardenContent = document.getElementById("gardenContent")
+
+skySelect.addEventListener("change", (event) => {
+    const skySelected = event.target.value;
+    console.log("Selected sky:", skySelected);
+    updateSky(skySelected);
+});
```

Comment:
A few things here: 
- The event handler I would put in its own function that why it would be easier to read as well as maintain.
- I would just moving the logic of adding event handlers to their respective HTML elements into something like `document.addEventListener("DOMContentLoaded", RegisterEventHandlers);`. That way we register the event handler functions to run when the DOM content has fully loaded. This ensures all HTML elements and resources are available before attempting to attach event handlers or manipulate the DOM.
- Don't forget to remove console logs like this, you could accidentally leak private data. Also just helps to keep the code clutter free.